frontend/src/hooks/useExceptionHandler.jsx

```diff
@@ -36,7 +34,6 @@ const useExceptionHandler = () => {
           }
           break;
         case "subscription_error":
-          navigate("/trial-expired");
```

Comment:
@tahierhussain Don't we still need to show trial expired if subscription error?

Reply:
@hari-kuriakose There is another bug here in front-end, because in code we are calling navigate via alert function the navigate added will not be triggered. This was noticed during test with @tahierhussain and is being taken up as a separate fix and PR.

@tahierhussain Will be moving the navigate and error capture in the axios interceptor for better handling of the same. cc: @vishnuszipstack